### PR TITLE
updates 'parse-link-header' dependency

### PR DESCRIPTION
**Pull request by jaredperreault-okta**
upgrades 'parse-link-header' dependency

OKTA-459293

### CODE REVIEW

File: test/it/factor-create.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ describe('Factors API', () => {
       factorType: 'call',
       provider: 'OKTA',
       profile: {
-        phoneNumber: '415 123 1234'
+        phoneNumber: '162 840 01133'
```
Comment (aarongranick-okta) [inline test/it/factor-create.ts:54]: why was this change needed? Was it to add the "1" (country code). If so can it be formatted as 

```
1 628 400 1133
```

Comment (jaredperreault-okta) [reply test/it/factor-create.ts:54]: Not entirely sure, but this is the value that is in master

**Review by shuowu-okta — APPROVED** [review]
